cloud_testing/xctest_parser/src/test/kotlin/xctest/ParseTest.kt

```diff
@@ -80,3 +123,9 @@ class ParseTest {
     }
 
 }
+
+fun ByteArray.md5(): String {
```

Comment:
for the download asset use case, we shouldn't need to calculate md5s

Reply:
Nope. You’re right. I was using this to sum the local file. Going off the file name is a better approach anyway.